### ISSUE TITLE
Pushing config updates to client using ActiveMQ(on AmazonMQ) and Spring cloud Config

### ISSUE THREAD

**Issue by madkt12** [spring-cloud/spring-cloud-bus#210]
Hi,
Recently I wanted to push real-time config updates to the client apps, I implemented it with rabbitMq and spring cloud bus and everything was ok. But because of some reason, I had to use ActiveMQ(on Amazon MQ). I searched a lot, unfortunately, couldn't find almost anything.

I wanted to ask you please could you tell me if there is any solution to this issue? If there isn't, what do you think if I start to implement it? Is it a good idea to do that?

Thanks

**Comment by madkt12** [comment]
Any update here?

**Comment by spencergibb** [comment]
Bus relies on spring cloud stream for binder implementations. Please ask there

**Comment by madkt12** [comment]
Hi,
I implemented a project using spring cloud config, spring cloud bus and Spring Cloud Stream binder ActiveMQ (Which I implemented it), feel free to take a look at them in the following repositories:
https://github.com/madkt12/spring-cloud-config-client
https://github.com/madkt12/spring-cloud-config-server
https://github.com/madkt12/spring-cloud-stream-binder-activemq

Best regards

**Comment by spencergibb** [comment]
The active mq binder is a question for spring cloud stream, not bus

**Comment by madkt12** [comment]
True, and I didn't repeat my question :)

As you already mentioned Spring cloud bus relies on Spring cloud Stream binder. 
I have implemented the binder for ActiveMQ, and the links that I shared here, is Spring Cloud Config (client and server) projects that are using Spring Cloud Config, Spring cloud bus and Spring Cloud Stream binder to push updated configuration to clients.

It might be helpful in case if someone reached here by googling.
